flang/lib/Lower/OpenMP/Utils.cpp

```diff
@@ -34,6 +34,22 @@ namespace Fortran {
 namespace lower {
 namespace omp {
 
+void checkAndApplyDeclTargetMapFlags(
+    Fortran::lower::AbstractConverter &converter,
+    llvm::omp::OpenMPOffloadMappingFlags &mapFlags,
+    const Fortran::semantics::Symbol &symbol) {
+  if (auto declareTargetOp =
+          llvm::dyn_cast_if_present<mlir::omp::DeclareTargetInterface>(
+              converter.getModuleOp().lookupSymbol(
+                  converter.mangleName(symbol)))) {
```

Comment:
Could you break it up into a few lines?

Reply:
Will do, although, I think this is a good example of reviewers having different tastes! ;-) I believe I was asked by @ergawy to collapse this into a single if in a prior iteration. I have no preference either way so happy to revert it to something more alike the the original form.

Reply:
In such cases use your best judgment.  IMO this looks horrible, but I'll respect your choice.  :)
Just respond with what you prefer.

Reply:
If Kareem complains, blame me.  :D

Reply:
I'm not a huge fan of the large one liner look either, I tend to think it's easier to approach/read things when they're a little more broken down into steps. But it's all preference at the end of the day and no one to blame!